source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,29 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	init: func (=_pointer, =_size) {
+	_owner: Bool
+	_forceFree := false
```

Comment:
It's not only for Recyclables though, it is used for Recoverables and any other ByteBuffer we might implement.